neural-engine/src/main/java/com/guardian/GuardianWorkflows/userworkflows/CreateNewUserRequest.java

```diff
@@ -3,17 +3,25 @@
 import java.util.HashMap;
 
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.PutItemRequest;
 
 public class CreateNewUserRequest {
 
     private int id;
     private String email;
+    private String firstName;
+    private String lastName;
+    private String password;
+    private String position;
 
-
-    public CreateNewUserRequest(int id, String email) {
-        if (isNullOrBlank(email)) throw new IllegalArgumentException("email cannot be null or blank");
+    public CreateNewUserRequest(int id, String email, String firstName, String lastName, String password, String position) {
+        if (isNullOrBlank(email) || isNullOrBlank(firstName) || isNullOrBlank(lastName) || isNullOrBlank(password) || isNullOrBlank(position)) throw new IllegalArgumentException("no entries can be null or blank");
```

Comment:
Would it be beneficial to create a variable for this conditional some like:

```
boolean FieldsMissingEntries = isNullOrBlank(email) || isNullOrBlank(firstName) || isNullOrBlank(lastName) || isNullOrBlank(password) || isNullOrBlank(position)

if(FieldsMissingEntries) throw new e
```

Non-blocking, just a thought.